cmd/swarmctl/service/flagparser/npipe.go

```diff
@@ -0,0 +1,35 @@
+package flagparser
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/docker/swarmkit/api"
+	"github.com/spf13/pflag"
+)
+
+// parseNpipe only supports a very simple version of anonymous npipes for
+// testing the most basic of data flows. Replace with a --mount flag, similar
+// to what we have in docker service.
+func parseNpipe(flags *pflag.FlagSet, spec *api.ServiceSpec) error {
+	if flags.Changed("npipe") {
+		npipes, err := flags.GetStringSlice("npipe")
+		if err != nil {
+			return err
+		}
+
+		container := spec.Task.GetContainer()
+
+		for _, npipe := range npipes {
+			if strings.Contains(npipe, ":") {
```

Comment:
@johnstep @cpuguy83 should we have more/other validation here? (don't know if there's a utility somewhere to validate these)

Reply:
@thaJeztah good question, there looks to be quite lot validations on engine side already. Is there is any point to duplicate them to here?